test/remote/gateways/remote_authorize_net_test.rb

```diff
@@ -384,10 +384,17 @@ def test_successful_authorization_with_moto_retail_type
   end
 
   def test_successful_verify
+    other_credit_card = credit_card('3088000000000017', brand: 'jcb')
+    response = @gateway.verify(other_credit_card, @options)
+    assert_success response
+    assert_equal 'This transaction has been approved', response.message
+    assert_equal response.responses.count, 2
+  end
+
+  def test_successful_verify_with_0_auth
```

Comment:
I think the names are switched? I'll fix this in the merge.

Reply:
code has been refactored, so please check again